scripts/setupSQLite3AsDatabaseDriver.st

```diff
@@ -0,0 +1,8 @@
+DatabaseAccessor classForThisPlatform DefaultDriver: SQLite3Driver.
+GlorpDatabaseLoginResource defaultLogin:
+  (Login new
+		database: UDBCSQLite3Platform new;
```

Comment:
Fold into SQLite3Platform at some point.  Leaving the UDBC prefix is confusing.

Reply:
Yes! I will do that kind of changes once the CI is setup for all the backends we wanted to test. So if we break something we will notice. :)